git_haiku.txt

```diff
@@ -0,0 +1,5 @@
+*Original Haiku by Liz*
+
+learning how to git
+it's not easy, it's really hard
```

Comment:
I think Haiku's have a 5-7-5 syllable pattern?  If so this line has one too many syllables.  

Let's change this to:
`it's not easy, it's complex`